src/adapters/store.js

```diff
@@ -119,3 +119,21 @@ export async function setLastLocation(loc) {
     throw e;
   }
 }
+
+export function getBetaPopupClosed() {
```

Comment:
I would move these function into `BetaInfoBox.jsx`.
IMO `store.js` should only expose a limited set of functions in order to abstract the way we store our persisted data (through localstorage).

If, at a later stage, we need to reuse these functions, then we will move theme into a new lib file like `betaPopup.js` for instance.

What do you think ?

Reply:
Sure, I did it like that to maintain the existing organization of store.js (containing all sorts of getters and setters for other components), but I can make something more generic, promise-less, and use it in my Alert.